libs/service-portal/core/src/lib/navigation/paths.ts

```diff
@@ -13,6 +13,12 @@ export enum ServicePortalPath {
 
   // Settings
   SettingsRoot = '/stillingar',
+  SettingsAccessControl = '/stillingar/adgangsstyring',
+  SettingsPersonalInformation = '/stillingar/personuupplysingar',
+  SettingsPersonalInformationEditPhoneNumber = '/stillingar/personuupplysingar/breyta-simanumeri',
+  SettingsPersonalInformationEditEmail = '/stillingar/personuupplysingar/breyta-netfangi',
+  SettingsPersonalInformationEditLanguage = '/stillingar/personuupplysingar/breyta-tungumali',
+  SettingsPersonalInformationEmailConfirmation = '/stillingar/personuupplysingar/stadfesta-netfang/:hash',
```

Comment:
The user profile service generates an email that has a hardcoded link to the old path, that will need to be updated as well. See: https://github.com/island-is/island.is/blob/main/apps/services/user-profile/src/app/user-profile/verification.service.ts

@obmagnusson Might have some more insights

Reply:
Great spot! That email should be using the navigation paths to prevent these errors from happening 👍 .. fixing it!

Reply:
Fixed.